File: src/Service/Vagon.java
package Service;

import EmployeeTasks.InterfazID;

public class Vagon implements InterfazID{
    
    private String idVagon;        
    private int capacidad;            

    public Vagon(String idVagon, int capacidad) {
        this.idVagon = idVagon;
        this.capacidad = capacidad;
    }

    @Override
    public String getId(){
        return idVagon;
    }

    public int getCapacidad() {
        return capacidad;
    }

}

File: Modulo5_vagones/Clases/Vagon.cs
using ProyectoEstructuras.Códigos.TareasEmpleado;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ProyectoEstructuras.Códigos.Servicio
{
    public class Vagon : InterfazID
    {
        private string idVagon;
        private int capacidad;

        public string IdVagon { get => idVagon; set => idVagon = value; }
        public int Capacidad { get => capacidad; set => capacidad = value; }

        public Vagon(string idVagon, int capacidad)
        {
            this.idVagon = idVagon;
            this.capacidad = capacidad;
        }

        public string GetId()
        {
            return IdVagon;
        }
    }
}
